content/en/docs/instrumentation/net/exporters.md

```diff
@@ -205,7 +205,8 @@ docker run \
 Next, install the Prometheus exporter:
 
 ```
-dotnet add package OpenTelemetry.Exporter.Prometheus
+dotnet add package OpenTelemetry.Exporter.Prometheus.AspNetCore --prerelease
+dotnet add package OpenTelemetry.Exporter.Prometheus.HttpListener --prerelease
```

Comment:
Why is this needed?

Reply:
This doc seems to be attempting to show both asp.net core app and non asp.net core app. If that is the intention, then need both packages. I think its best to stick with asp.net core here,. (the non asp.net core one is anyway not prodcution recommended)

@markpitt 
In the aspnetcore part, the doc is missing important step : https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.Prometheus.AspNetCore#step-3-configure-prometheus-scraping-endpoint

Reply:
@cijothomas Thanks for the link. I'd been looking for some more detail on how it's configured. 

I've added the the registration of the middleware and removed the HttpClient package. Hopefully, this fixes all the issues